server/datastore/mysql/vpp.go

```diff
@@ -651,11 +651,10 @@ func (ds *Datastore) getOrInsertSoftwareTitleForVPPApp(ctx context.Context, tx s
 			selectStmt = `
 				    SELECT id
 				    FROM software_titles
-				    WHERE (bundle_identifier = ? OR (name = ? AND browser = ''))
-				      AND source NOT IN ('ios_apps', 'ipados_apps')
+				    WHERE bundle_identifier = ? AND additional_identifier = 0
 				    ORDER BY bundle_identifier = ? DESC
```

Comment:
We can drop the order by desc limit 1 now because we are querying on a unique index. Sorry for missing that previously.

Reply:
np i missed it too!